package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1711,7 +1711,87 @@ def radius_of_gyration(group, wrap=False, **kwargs):
     @warn_if_not_unique
     @_pbc_to_wrap
     @check_atomgroup_not_empty
-    def shape_parameter(group, wrap=False):
+    def gyration_moments(group, wrap=False, unwrap=None, compound='group'):
+        """Moments of the gyration tensor.
```

Comment:
Could you please provide an equation and paper describing the property? I and users would benefit from more detailed docs. :)